cms-publish-rest/src/main/java/se/simonsoft/cms/publish/rest/PublishRelease.java

```diff
@@ -15,30 +15,31 @@
  */
 package se.simonsoft.cms.publish.rest;
 
-import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
 
 import se.simonsoft.cms.item.CmsItem;
 import se.simonsoft.cms.publish.config.databinds.config.PublishConfig;
 import se.simonsoft.cms.publish.config.databinds.profiling.PublishProfilingRecipe;
+import se.simonsoft.cms.reporting.response.CmsItemRepositem;
 
 import java.util.Map;
 import java.util.Set;
 
-@JsonIgnoreProperties(ignoreUnknown = true)
 public class PublishRelease {
 
-    private CmsItem item;
+    private CmsItemRepositem item;
     private Map<String, PublishConfig> config;
     private Map<String, PublishProfilingRecipe> profiling;
-    Map<String, Set<String>> translationExecutions;
-    Map<String, Set<String>> releaseExecutions;
+    private Map<String, Set<String>> translationExecutions;
```

Comment:
I don't think execution status should be in this class. This class will only be requested once when loading the react form.

Status should be requested on interval from the existing service:  'publish4/release/status'

Reply:
They are hidden from the serialized JSON output and only exist there to keep it cleaner and to be able to reuse the PublishRelease for both the HTML and the JSON outputs. I can figure something else if you don't like it.

Reply:
Ok, I think the HTML output will be simplified to only a title and the "react element"?

Reply:
True, and the navigation bar elements. I shall cleanup and remove what's not used. Any wishes where to place the template?